10_Examples/will_onboarding1/include/will_onboarding1/Mather.hpp

```diff
@@ -0,0 +1,37 @@
+#pragma once
+
+#include <chrono>
+#include <memory>
+#include <string>
+
+#include "rclcpp/rclcpp.hpp"
+#include "std_msgs/msg/int32.hpp"
+
+using namespace std::chrono_literals;
+using namespace std;
+using std::placeholders::_1;
+
+namespace Math_Space {
+
+class Mather : public rclcpp::Node {
+public:
+	Mather(int a, int b);
+
+	int decode(int encoded);
+
+	void topic_callback(const std_msgs::msg::Int32::SharedPtr msg);
+
+	void timer_callback();
+
+private:
+	int a;
+	int b;
+	int decodeInt;
```

Comment:
member variables are always named either m_thing, or thing_. 

Which one depends on who you are working with. I do m_ these days, but may have been trailing underscore when some of GHOST code was written.